noxfile.py

```diff
@@ -0,0 +1,69 @@
+from shutil import rmtree
+from textwrap import dedent
+from pathlib import Path
+import nox
+
+ROOT = Path(__file__).parent
+STYLEGUIDE = ROOT / "styleguide"
+SRC_DIRECTORY = STYLEGUIDE
+BUILD_DIRECTORY = STYLEGUIDE / "_build"
+
+nox.options.sessions = ['build', 'open']
+
+
+@nox.session(python=None)
+def build(session):
+    """Build the styleguide."""
+    session.run(
+        "sphinx-build",
+        f"{SRC_DIRECTORY.resolve()}",
+        f"{(BUILD_DIRECTORY / 'html').resolve()}",
+        external=True
+    )
+
+
+@nox.session(python=None)
+def clean(session):
+    """Remove the build directory."""
+    if BUILD_DIRECTORY.exists():
+        rmtree(BUILD_DIRECTORY.resolve())
+        session.log(f"Removed {BUILD_DIRECTORY}")
+
+
+@nox.session(python=None)
+def open(session):
+    """Open the styleguide in the browser."""
+    index_page = BUILD_DIRECTORY / "html" / "index.html"
+    if not index_page.exists():
+        session.error(
+            dedent(
+            f"""
+            File {index_page} does not exist.
+            Please run `nox -s build` first
+            """
+            )
+        )
+    session.run(
+        "python",
+        "-m",
+        "webbrowser",
```

Comment:
interesting

Reply:
The python interpreter/modules ship with quite cool `cli-tools` here are two more cool ones:
* `python -m json.tool -h`
* `python -m zipfile -h`

usually you can find at the bottom of the documentation for a specific module, if it has some sort of `cli`/`commands`
e.g. https://docs.python.org/3/library/zipfile.html#command-line-interface